x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-final_pipeline.json

```diff
@@ -65,7 +72,8 @@
     }
   ],
   "_meta": {
-    "managed": true
+    "managed": true,
+    "geoip_database_lazy_download": true
```

Comment:
ℹ️ Adding `geoip_database_lazy_download` to our pipeline, so the database is downloaded only when an Analytics Collection is created.